tgui/packages/tgui/interfaces/LobbyMenu.tsx

```diff
@@ -0,0 +1,608 @@
+                          <Box>
+                            <Stack vertical>
+                              <Stack.Item>
+                                Are you sure want to attempt joining as a
+                                Xenomorph?
+                              </Stack.Item>
+                            </Stack>
+                            <Stack justify="center">
+                              <Stack.Item>
+                                <Button onClick={() => act('late_join_xeno')}>
+                                  Confirm
+                                </Button>
+                              </Stack.Item>
+                            </Stack>
+                          </Box>
+                        </ModalConfirm>,
+                      );
+                    }}
+                  >
+                    Join the Hive
+                  </LobbyButton>
+                </Stack.Item>
+                <Stack.Item>
+                  <LobbyButton
+                    icon="users-rays"
+                    tooltip="View Hive Leaders"
+                    index={9}
+                    onClick={() => act('hiveleaders')}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+            {!!upp_enabled && (
+              <Stack.Item>
+                <LobbyButton
+                  index={7}
+                  onClick={() => act('late_join_upp')}
+                  icon="users-between-lines"
+                >
+                  Join the UPP
+                </LobbyButton>
+              </Stack.Item>
+            )}
+            {!!predator_enabled && (
+              <Stack.Item>
+                <LobbyButton
+                  index={7 + (upp_enabled ? 1 : 0)}
+                  onClick={() => {
+                    setModal(
+                      <ModalConfirm>
+                        <Box>
+                          <Stack vertical>
+                            <Stack.Item>
+                              Are you sure want to attempt joining as a
+                              Predator?
+                            </Stack.Item>
+                          </Stack>
+                          <Stack justify="center">
+                            <Stack.Item>
+                              <Button onClick={() => act('late_join_pred')}>
+                                Confirm
+                              </Button>
+                            </Stack.Item>
+                          </Stack>
+                        </Box>
+                      </ModalConfirm>,
+                    );
+                  }}
+                >
+                  <Flex>
+                    <Flex.Item>
+                      <Box className="gnarp" inline />
+                    </Flex.Item>
+                    <Flex.Item>Join the Hunt</Flex.Item>
+                  </Flex>
+                </LobbyButton>
+              </Stack.Item>
+            )}
+            {!!fax_responder_enabled && (
```

Comment:
Same thing here, this should just be a boolean value at all times.
If it can be undefined, declare a default value

Reply:
nvm this is a byond boolean so this is fine